components/Form.js

```diff
@@ -37,7 +42,7 @@ export default function Form({ currentTrips, setCurrentTrips }) {
         Start date:
         <StyledInput
           type="date"
-          name="date"
+          name="startdate"
           placeholder="Choose a start date"
```

Comment:
I had a tremendous stress going through the QA without a type of restriction or validation from user inputs fields. Is this planned in future?